.circleci/continue_config.yml

```
@@ -0,0 +1,284 @@
+          # "The Windows executor currently only supports Windows containers.
+          # Running Linux containers on Windows is not possible for now"
+          # (from https://circleci.com/docs/2.0/hello-world-windows/)
+          name: Run unit tests
+          command: |
+            conda activate kedro_plugins
+            pytest <<parameters.plugin>>\tests
+
+jobs:
+  unit_tests:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    machine:
+      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
+      image: circleci/classic:201711-01
+      docker_layer_caching: true
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+      - run:
+          name: Run tests
+          command: |
+            make plugin=<<parameters.plugin>> test
+
+  e2e_tests:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    machine:
+      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
+      image: circleci/classic:201711-01
+      docker_layer_caching: true
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+      - run:
+          name: Run all end to end tests
+          command: |
+            make plugin=<<parameters.plugin>> e2e-tests
+
+  lint:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    machine:
+      # Don't use 2018 image: https://discuss.circleci.com/t/24639/18
+      image: circleci/classic:201711-01
+      docker_layer_caching: true
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - setup:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+      - run:
+          name: Run pylint and flake8
+          command: |
+            make plugin=<<parameters.plugin>> lint
+
+  # Windows jobs
+  win_build:
+    parameters:
+      python_version:
+        type: string
+      plugin:
+        type: string
+    executor:
+      name: win/default
+    working_directory: ~/repo
+    environment:
+      python_version: <<parameters.python_version>>
+    steps:
+      - win_tests:
+          python_version: <<parameters.python_version>>
+          plugin: <<parameters.plugin>>
+
+
+workflows:
+  # when pipeline parameter, run-build-kedro-telemetry is true, the
+  # kedro-telemetry job is triggered.
+  kedro-telemetry:
+    when: << pipeline.parameters.run-build-kedro-telemetry >>
+    jobs:
+      - unit_tests:
+          matrix:
+            parameters:
+              python_version: [ "3.6", "3.7", "3.8" ]
+              plugin: [ "kedro-telemetry" ]
```

```suggestion
              python_version: ["3.6", "3.7", "3.8"]
              plugin: ["kedro-telemetry"]
```
Don't know why but I find all all these spaces upsetting 😅  Obviously just nitpicking though...